### ISSUE TITLE
Linux version

### ISSUE THREAD

**Issue by jcblw** [jcblw/pastila#36]
Does not seems to run on linux 

```
A JavaScript error occured in the browser process
Uncaught Exception:
LevelUPError: Could not locate LevelDOWN, try `npm install leveldown`
    at getLevelDOWN (/home/jacob/Projects/Apps/pastila/app/node_modules/levelup/lib/util.js:71:11)
    at LevelUP.open (/home/jacob/Projects/Apps/pastila/app/node_modules/levelup/lib/levelup.js:113:37)
    at new LevelUP (/home/jacob/Projects/Apps/pastila/app/node_modules/levelup/lib/levelup.js:86:8)
    at LevelUP (/home/jacob/Projects/Apps/pastila/app/node_modules/levelup/lib/levelup.js:46:12)

```

**Comment by jcblw** [comment]
this seems related #40 need to retest using `iojs 1.2.0` pretty sure this might already be resolved

**Comment by jcblw** [comment]
cool! so just tested out on a ubuntu box and this seems to be resolved installed everything using version `4.1.2` version of nodejs and everthing seems to be fine. The fonts are off but not a huge deal in terms of functionality.
